src/app/pages/search/decision-support-interventions/decision-support-interventions-view.jsx

```diff
@@ -0,0 +1,307 @@
+              && (
+                <>
+                  <TableContainer className={classes.tableContainer} component={Paper}>
+                    <Table
+                      stickyHeader
+                      aria-label="Decision Support Interventions Search table"
+                    >
+                      <ChplSortableHeaders
+                        headers={headers}
+                        onTableSort={handleTableSort}
+                        orderBy={orderBy}
+                        order={sortDescending ? 'desc' : 'asc'}
+                        stickyHeader
+                      />
+                      <TableBody>
+                        { listings
+                          .map((item) => (
+                            <TableRow key={item.id}>
+                              <TableCell className={classes.stickyColumn}>
+                                <strong>
+                                  <ChplLink
+                                    href={`#/listing/${item.id}`}
+                                    text={item.chplProductNumber}
+                                    analytics={{
+                                      ...analytics,
+                                      event: 'Navigate to Listing Details Page',
+                                      label: item.chplProductNumber,
+                                      aggregationName: item.product.name,
+                                    }}
+                                    external={false}
+                                    router={{ sref: 'listing', options: { id: item.id } }}
+                                  />
+                                </strong>
+                              </TableCell>
+                              <TableCell>
+                                <ChplLink
+                                  href={`#/organizations/developers/${item.developer.id}`}
+                                  text={item.developer.name}
+                                  analytics={{
+                                    ...analytics,
+                                    event: 'Navigate to Developer Page',
+                                    label: item.developer.name,
+                                  }}
+                                  external={false}
+                                  router={{ sref: 'organizations.developers.developer', options: { id: item.developer.id } }}
+                                />
+                              </TableCell>
+                              <TableCell>{item.product.name}</TableCell>
+                              <TableCell>{item.version.name}</TableCell>
+                              <TableCell>{ getStatusIcon(item.certificationStatus) }</TableCell>
+                              <TableCell className={classes.linkWrap}>
+                                { item.riskManagementSummaryInformationValue
+                                  ? (
+                                    <dl>
```

Comment:
I think the `dl - dd` construct was for the API Information page where there could have been multiple values. For this one, where there's only one option, we don't need the list